### DISCUSSION TITLE
Ticket for Profile help open 38 days

### DISCUSSION THREAD

**Discussion by stevenrobson** [community/community#56173] [Other Feature Feedback, Questions, & Ideas]
Is it common for support tickets with GitHub to be open for over a month without a single response? How many months should I expect before hearimg from support?

**Comment by Zippyflash62** [comment]
Im trying to find my position on github. Ill answer that a ticket should bé answered no more than a day.  Are yôu asking me. Who is the admin?

**Comment by rskumar** [comment]
It's disappointing to see that production was down, team was not able to access Wiki, due to failed payment. No urgency is taken in account for even team (paid) plan. Ticket is open for more than 30 hours, without any comment from support side.
